package-lock.json

```diff
@@ -5759,7 +5757,6 @@
           "version": "5.0.0",
           "resolved": "https://registry.npmjs.org/@types/bignumber.js/-/bignumber.js-5.0.0.tgz",
           "integrity": "sha512-0DH7aPGCClywOFaxxjE6UwpN2kQYe9LwuDQMv+zYA97j5GkOMo8e66LYT+a8JYU7jfmUFRZLa9KycxHDsKXJCA==",
-          "optional": true,
```

Comment:
I do not understand exactly why this happens or what the implications are. I don't think this change is a problem, nor is there any way to really avoid this kind of stuff with `npm` as it is today, just curious why.